src/stores/ForeignStore.js

```diff
@@ -126,6 +128,11 @@ class ForeignStore {
       } catch(e) {
         this.tokenName = this.foreignWeb3.utils.hexToAscii(await getName(alternativeContract)).replace(/\u0000*$/, '')
       }
+      try {
+        this.tokenDecimals = await getDecimals(this.tokenContract)
+      } catch(e) {
+        this.tokenDecimals = this.foreignWeb3.utils.hexToAscii(await getDecimals(alternativeContract)).replace(/\u0000*$/, '')
```

Comment:
There is no need to call `getDecimals` with `alternativeContract`. It is only needed for `name` and `symbol`

Reply:
ok I remove the alternativeContract call in https://github.com/poanetwork/bridge-ui/pull/165/commits/666906774c93e0b1205a327b589696aaadfead29